test/controls/color-picker.test.js

```diff
@@ -11,14 +11,6 @@ test('ColorPicker toggles expanded when swatch is clicked', () => {
   expect(wrapper.find('.popover').exists()).toBe(false)
 })
 
-test('ColorPicker collapses when background is clicked', () => {
-  const wrapper = mount(<ColorPicker />)
-  wrapper.find('.swatch').simulate('click')
-  expect(wrapper.find('.popover').exists()).toBe(true)
-  wrapper.find('.cover').simulate('click')
-  expect(wrapper.find('.popover').exists()).toBe(false)
-})
```

Comment:
Enzyme's `click` simulation doesn't trigger the `OnOutsideClick` component's logic. I'm _ok_ with that, given that we're using a third party library which has its own test coverage